mslib/msui/mscolab.py

```diff
@@ -163,6 +163,8 @@ def disconnect_handler(self):
         self.loginButton.setEnabled(False)
         self.addUser.setEnabled(False)
         self.connectMscolab.setEnabled(True)
+        self.url.setEnabled(True)
+        self.url.setEditable(True)
```

Comment:
Not sure but do we need to use setEditable as well? Won't just using setEnabled(False) be enough as it completely disabled the input?

Reply:
Yes you're right. I did not think of that. Wouldn't need line 167 as well if setEnabled is True!